packages/shared/src/components/profile/TopReaderWidget.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { ReactElement } from 'react';
+import {
+  ActivityContainer,
+  ActivitySectionHeader,
+  ActivitySectionSubTitle,
+} from './ActivitySection';
+import { topReaderBadgeDocs } from '../../lib/constants';
+import { MedalBadgeIcon } from '../icons';
+import { IconSize } from '../Icon';
+import { BadgeIconGoldGradient } from '../badges/BadgeIcon';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../typography/Typography';
+import { formatDate, TimeFormatType } from '../../lib/dateFormat';
+import type { PublicProfile } from '../../lib/user';
+import { ClickableText } from '../buttons/ClickableText';
+import { useTopReader } from '../../hooks/useTopReader';
+
+export const TopReaderWidget = ({
+  user,
+}: {
+  user: PublicProfile;
+}): ReactElement => {
+  const { data: topReaders, isPending: isTopReaderLoading } = useTopReader({
```

Comment:
was going to suggest we could still expose `isLoading` from the hook, but actually think this is better for everyone to know which one it actually is :)